hack/lint-all-zarf-packages.sh

```diff
@@ -1,4 +1,6 @@
-#!/bin/bash
+#!/usr/bin/env bash
+
+set -euo pipefail
 
 ZARF_BIN=$1
 LINT_SRC_TEST=$2
```

Comment:
<img width="1568" alt="unbound-var" src="https://github.com/defenseunicorns/zarf/assets/87675701/1913624a-e957-4aac-9283-888c49cba399">

When this script is called in `make lint-packages-and-examples`, it fails because we do not provide a value for the `LINT_SRC_TEST` variable